nvme-ioctl.c

```diff
@@ -70,7 +70,7 @@ int nvme_ns_rescan(int fd)
 
 int nvme_get_nsid(int fd)
 {
-	static struct stat nvme_stat;
```

Comment:
I don't get why we need to update the global nvme_stat, and what the problem is. We need to have it in the commit message.